subsystem/arm.py

```diff
@@ -0,0 +1,46 @@
+import config
+import constants
+
+from units.SI import radians
+from toolkit.motors.ctre_motors import TalonFX
+from toolkit.subsystem import Subsystem
+
+class Arm(Subsystem):
+
+    # Initialize class
+    def __init__(self):
+        super().__init__()
+        self.arm_motor = TalonFX(can_id=constants.ARM_MOTOR_ID, config=config.arm_config)
+        self.arm_motor_follower = TalonFX(can_id=constants.ARM_MOTOR_ID, inverted=True, config=config.arm_config)
+
+        self.zeroed = False
+        self.arm_moving = False
+
+    # Start motors
+    def init(self) -> None:
+        self.arm_motor.init()
+        self.arm_motor_follower.init()
+        self.arm_motor_follower.follow(self.arm_motor)
+
+    # Zero the arm
+    def zero(self) -> None:
+        self.arm_motor.set_target_position(self.arm_motor.get_sensor_position() * constants.arm_gear_ratio)
+        self.zeroed = True
+
+    # Extends arm to an radians
+    def extend(self, radians: radians) -> None:
```

Comment:
ensure that the angle is between the min and max value of the arm before setting angle